### PR TITLE
MCOL-5597 Rollback changes

### PR DESCRIPTION
**Pull request by denis0x0D**

### CODE REVIEW

**Review by mariadb-SergeyZefirov — APPROVED** [review]
I approve these changes in general, as a palliative measue. This is the best we can do in our current circumstances.

File: utils/joiner/joinpartition.cpp [hunk 1]
```diff
@@ -481,8 +486,8 @@ int64_t JoinPartition::processSmallBuffer(RGData& rgData)
 
     ret = writeByteStream(0, bs);
 
-    if (rg.getRowCount())
-      htSizeEstimate += rg.getDataSize();
+    // FIXME(MCOL-5597): Properly calculate the size of the bucket.
```
Comment (drrtuy) [inline utils/joiner/joinpartition.cpp:489]: Are you sure this is the properly calculated size of the RG ? 
I mean the wording doesn't express the fact that Stringstore size is ignored. I suggest to elaborate on the fact that this is a heuristict and we ignore the Stringstore b/c the tests tells us to do so.

**Review by drrtuy — CHANGES_REQUESTED** [review]
Plz comment on the note I left in the patch.

File: utils/joiner/joinpartition.cpp (hunk 1 above)
Comment (denis0x0D) [reply utils/joiner/joinpartition.cpp:489]: Right, it's not a proper calculation and tag "FIXME" is added to specify this.
The reason of this patch  - @mariadb-LeonidFedorov asked to apply rollback based on @mariadb-SergeyZefirov research to pass 5597

Comment (drrtuy) [reply utils/joiner/joinpartition.cpp:489]: Ah, I misinterpreted `FIXME` part. 
Could you describe this FIXME in more words?

**Review by drrtuy — APPROVED** [review]
untested ACK

**Pull request merged**